cle/backends/soot.py

```diff
@@ -0,0 +1,180 @@
+
+import zipfile
+
+import archinfo
+from archinfo.arch_soot import SootMethodDescriptor, SootAddressDescriptor
+from pysoot.lifter import Lifter
+
+from . import Backend
+from . import register_backend
+from ..errors import CLEError
+
+import logging
+l = logging.getLogger("cle.backends.soot")
+
+
+class Soot(Backend):
+
+    def __init__(self, path, additional_jars=None, additional_jar_roots=None, main_class=None, *args, **kwargs):
+
+        if 'has_memory' in kwargs and kwargs['has_memory'] is not False:
```

Comment:
wouldn't this be nicer as either a default kwarg or a `kwargs.get('has_memory', False)`?